build.sbt

```diff
@@ -62,39 +62,47 @@ lazy val microservice = Project(appName, file("."))
   .enablePlugins(play.sbt.PlayScala, SbtAutoBuildPlugin, SbtGitVersioning, SbtDistributablesPlugin)
   .disablePlugins(sbt.plugins.JUnitXmlReportPlugin)
   .disablePlugins(JUnitXmlReportPlugin)
-  .settings(
-    // To resolve a bug with version 2.x.x of the scoverage plugin - https://github.com/sbt/sbt/issues/6997
-    libraryDependencySchemes ++= Seq("org.scala-lang.modules" %% "scala-xml" % VersionScheme.Always,
-    )
-  )
   .settings(PlayKeys.playDefaultPort := 10005)
   .settings(
     majorVersion := 0,
     scalaVersion := "2.13.12",
```

Comment:
Could you please add: 
`ThisBuild / majorVersion := 0
ThisBuild / scalaVersion := "2.13.12"`

to the top of the file and remove the settings for this in here and also for it.